tests/test_scheduler.py

```diff
@@ -72,7 +72,7 @@ def test_schedule(self):
             # the timestamp maybe different
             registry.schedule(job, datetime(2019, 1, 1))
             self.assertEqual(self.testconn.zscore(registry.key, job.id),
-                             1546300800 + time.timezone)  # 2019-01-01 UTC in Unix timestamp
+                             1546300800 + (time.timezone if time.daylight == 0 else time.altzone))  # 2019-01-01 UTC in Unix timestamp
```

Comment:
Do you mind improving the test to show that it works properly when daylight saving is both on and off? We should also change the assertion to either use `time.altzone` or `time.timezone` to make sure the result is correct.